feature-libs/asm/components/asm-bind-cart/asm-bind-cart.component.ts

```diff
@@ -70,26 +83,57 @@ export class AsmBindCartComponent implements OnInit, OnDestroy {
   );
 
   activeCartId = '';
+  inactiveCartId = '';
+  isInactiveCartSaved = false;
 
   @ViewChild('bindToCart') bindToCartElemRef: ElementRef<HTMLButtonElement>;
+  @ViewChild('saveInactiveCart')
+  saveInactiveCartElemRef: ElementRef<HTMLButtonElement>;
 
   protected subscription = new Subscription();
 
+  constructor(
+    globalMessageService: GlobalMessageService,
+    activeCartFacade: ActiveCartFacade,
+    multiCartFacade: MultiCartFacade,
+    asmBindCartFacade: AsmBindCartFacade,
+    launchDialogService: LaunchDialogService,
+    savedCartFacade: SavedCartFacade,
+    // eslint-disable-next-line @typescript-eslint/unified-signatures
+    asmComponentService: AsmComponentService,
+    routing: RoutingService,
+    featureConfig: FeatureConfigService
+  );
+  constructor(
```

Comment:
Could you please add a comment as follow ?
  /**
   * @deprecated since 7.0
   */